### DISCUSSION TITLE
extremely slow uploads

### DISCUSSION THREAD

**Discussion by 111100001** [bibanon/tubeup#323] [General]
is this normal?, would rerunning the script solve it? it doesnt show s3 rate limiting but very slow

**Comment by brandongalbraith** [comment]
tubeup is only a tool, and has no control over Internet Archive ingest speeds. They can and do rate limit and throttle uploads, depending on content and uploader.

**Comment by vxbinaca** [comment]
> is this normal?, would rerunning the script solve it? it doesnt show s3 rate limiting but very slow

Modify your behavior, we can't change S3 rate limiting thats IA. And no we can't enable you to bypass it.
